kotlin/internal/toolchains.bzl

```diff
@@ -131,6 +131,7 @@ _kt_toolchain = rule(
                 "1.6",
                 "1.7",
                 "1.8",
+                "1.9",
```

Comment:
We technically need to include 2.0 here now.

Kotlin 1.9 has deprecated a lot of the K2 related flags and are now asking developers to use `language_version = 2.0` to enable the K2 compiler.

Reply:
Which means that this opt is a no-op now https://github.com/bazelbuild/rules_kotlin/blob/f229f1ce273f63b61f072c1304d2f5efb677bdd4/src/main/starlark/core/options/opts.kotlinc.bzl#L245-L256